lib/kubernetes-deploy/kubernetes_resource/deployment.rb

```diff
@@ -14,6 +14,11 @@ def sync
         @rollout_data = { "replicas" => 0 }.merge(deployment_data["status"]
           .slice("replicas", "updatedReplicas", "availableReplicas", "unavailableReplicas"))
         @status = @rollout_data.map { |state_replicas, num| "#{num} #{state_replicas.chop.pluralize(num)}" }.join(", ")
+        deployment_data["status"]["conditions"].each do |condition|
```

Comment:
I think this could be shortened to:
`@progress = deployment_data["status"]["conditions"].find { |condition| condition['type'] == 'Progressing' }`